File: fizzbuzz.cpp
#include <iostream>

int main() {
  int fizz, buzz, lines;
  std::cin >> fizz >> buzz >> lines;

  for (int i = 1; i <= lines; i++) {
    if ((i % fizz) == 0 && (i % buzz) == 0) {
      std::cout << "FizzBuzz\n";
    }
    else if ((i % fizz) == 0) {
      std::cout << "Fizz\n";
    }
    else if ((i % buzz) == 0) {
      std::cout << "Buzz\n";
    }
    else {
      std::cout << i << "\n";
    }
  }
  return 0;
}

File: fizzbuzz.py
inline = input().split()
fizz = int(inline[0])
buzz = int(inline[1])
lines = int(inline[2])

for i in range(1,lines+1):
    if i % fizz == 0 and i % buzz == 0:
        print("FizzBuzz")
    elif i % fizz == 0:
        print("Fizz")
    elif i % buzz == 0:
        print("Buzz")
    else:
        print(i)
